lib/load-json.js

```diff
@@ -66,12 +66,10 @@ const getJsonSrc = sourceObj => {
       }
     }))
   }
-  let jsonPaths
+  let jsonPaths = []
   try {
-    let testPath = fs.readdirSync(sourceObj.path)
-    if (testPath) {
-      jsonPaths = glob.sync(`${sourceObj.path}/**/*.json`)
-    }
+    fs.readdirSync(sourceObj.path)
```

Comment:
Without assigning it to a var, it feels a little side-effecty? Wonder if `fs.stat` would be more semantically correct? (in that here, we're not really reading a directory)

Reply:
If we assign to a var 

this code is as is - the testPath condition was redundant

Semantically it's probably better as `fs.existsSync` which has been undeprecated. So I've switch to that